server/chat.ts

```diff
@@ -989,7 +989,7 @@ export class CommandContext extends MessageContext {
 				}
 				if (targetUser.settings.blockPMs &&
 					(targetUser.settings.blockPMs === true || !user.authAtLeast(targetUser.settings.blockPMs)) &&
-					!user.can('lock')) {
+					(!user.can('lock') || user.group === '*') {
```

Comment:
```suggestion
					(!user.can('lock') || user.group === '*')) {
```
that's my fault, I'm bad at reading :)

Reply:
Global bots can't lock, though...

Reply:
Is this actually the check? If so, in theory, nothing else needs to be done, bot PMs are already blockable.

Reply:
I can confirm that global bots don't bypass blocked PMs on main. This PR can probably be closed along with the suggestion thread.